_data-prepper/common-use-cases/s3-logs.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: default
+title: S3 logs
+parent: Common use cases
+nav_order: 20
+---
+
+# S3 logs
+
+Data Prepper allows you to load logs from [Amazon Simple Storage Service](https://aws.amazon.com/s3/) (Amazon S3), including traditional logs, JSON documents, and CSV logs.
+
+
+## Architecture
+
+Data Prepper can read objects from S3 buckets using an [Amazon Simple Queue Service (SQS)](https://aws.amazon.com/sqs/) (Amazon SQS) queue and [Amazon S3 Event Notifications](https://docs.aws.amazon.com/AmazonS3/latest/userguide/NotificationHowTo.html).
+
+Data Prepper polls the Amazon SQS queue for S3 event notifications. When Data Prepper receives a notification that an S3 object was created, Data Prepper reads and parses that S3 object.
+
+The following diagram shows the overall architecture of the components involved.
+
+<img src="{{site.url}}{{site.baseurl}}/images/data-prepper/s3-source/s3-architecture.jpg" alt="S3 source architecture">{: .img-fluid}
+
+The flow of data is as follows.
+
+1. A system produces logs into the S3 bucket.
+2. S3 creates an S3 event notification in the SQS queue.
+3. Data Prepper polls Amazon SQS for messages and then receives a message.
+4. Data Prepper downloads the content from the S3 object.
+5. Data Prepper sends a document to OpenSearch for the content in the S3 object.
+
+
+## Pipeline overview
+
+Data Prepper supports reading data from S3 using the [`s3` source]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/sources/s3/).
+
+The following diagram shows a conceptual outline of a Data Prepper pipeline reading from S3.
+
+<img src="{{site.url}}{{site.baseurl}}/images/data-prepper/s3-source/s3-pipeline.jpg" alt="S3 source architecture">{: .img-fluid}
+
+## Prerequisites
+
+Before Data Prepper can read log data from S3, you need the following prerequisites: 
+
+- An S3 bucket.
+- A log producer that writes logs to S3. The exact log producer will vary depending on your specific use case, but could include writing logs to S3 or a service such as Amazon CloudWatch.
+
+
+## Getting started
+
+Use the following steps to begin loading logs from S3 with Data Prepper.
+
+1. Create an [SQS standard queue](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/step-create-queue.html) for your S3 event notifications. 
+2. Configure [bucket notifications](https://docs.aws.amazon.com/AmazonS3/latest/userguide/ways-to-add-notification-config-to-bucket.html) for SQS. Use the `s3:ObjectCreated:*` event type.
+3. Grant [AWS IAM](https://docs.aws.amazon.com/IAM/latest/UserGuide/introduction.html) permissions to Data Prepper for accessing SQS and S3.
```

Comment:
IAM needs to be defined on first appearance: AWS Identity and Access Management (IAM). Just IAM on subsequent appearance.